backend/functions/lib/adapters/gateways/dxtr/orders.dxtr.gateway.test.ts

```diff
@@ -11,7 +11,15 @@ import {
 import { ApplicationContext } from '../../types/basic';
 import { MockData } from '../../../../../../common/src/cams/test-utilities/mock-data';
 
-const dxtrCaseDocketEntries: DxtrOrderDocketEntry[] = [
+function getEarliestDate(docket: DxtrOrderDocketEntry[]) {
+  const earliestDate = docket.reduce<string>((earliestDate, de) => {
+    if (!earliestDate || earliestDate > de.dateFiled) return de.dateFiled;
+    return earliestDate;
+  }, null);
+  return earliestDate;
+}
```

Comment:
I would avoid using the same variable name for the returned value and the variable inside the reduce function.  This could be a bit confusing.

Reply:
Better. We can just return the output of the reduce function and remove the variable declaration altogether.